packages/web/src/common/store/pages/signon/sagas.ts

```diff
@@ -459,44 +589,105 @@ function* signUp() {
   const handle = signOn.handle.value
   const alreadyExisted = signOn.accountAlreadyExisted
   const referrer = signOn.referrer
-  const createUserMetadata = {
-    name,
-    handle,
-    profilePicture: (signOn.profileImage?.file as File) || null,
-    coverPhoto: (signOn.coverPhoto?.file as File) || null,
-    isVerified: signOn.verified,
-    location
-  }
 
   yield* call(audiusBackendInstance.setUserHandleForRelay, handle)
 
-  const feePayerOverride = yield* select(getFeePayer)
-
   yield* put(
     confirmerActions.requestConfirmation(
       handle,
-      function* () {
+      async function* () {
```

Comment:
why is this async needed?

Reply:
Oh it's not. Good catch. I'm not sure how that got there.